ndcube/ndcube.py

```diff
@@ -1458,6 +1458,25 @@ def fill_masked(self, fill_value, uncertainty_fill_value=None, unmask=False, fil
             self.mask = False
         return None
 
+    @property
+    def crop_input_types_order(self):
+        """
+        Returns types and order of high-level coordinate objects that must be input to .crop()
+        Each point given to .crop() must be a tuple of scalar high-level coordinate objects
+        of these types, in this order.  Note, however, that any of these can be replaced
+        with None if cropping along the axis/axes of that world type in not desired.
```

Comment:
First line of the docstring should be <80 char. Then a blank line, then a paragraph. (Same below)

Reply:
Also put code snippets in double backticks.